testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/TestHiveStorageFormats.java

```diff
@@ -254,6 +263,22 @@
         };
     }
 
+    @DataProvider
+    public static StorageFormat[] storageFormatsWithZeroByteFile()
+    {
+        return new StorageFormat[] {
+                storageFormat("ORC"),
+                storageFormat("PARQUET"),
+                storageFormat("RCBINARY"),
+                storageFormat("RCTEXT"),
+                storageFormat("SEQUENCEFILE"),
+                storageFormat("TEXTFILE"),
+                storageFormat("AVRO"),
+                storageFormat("CSV"),
+                storageFormat("JSON")
```

Comment:
Any idea why we don't include "CSV" and "JSON" in the existing "storageFormats" dataProvider?

Reply:
```
        Set<String> allFormatsToTest = allFormats.stream()
                // Hive CSV storage format only supports VARCHAR, so needs to be excluded from any generic tests
                .filter(format -> !"CSV".equals(format))
                // TODO when using JSON serde Hive fails with ClassNotFoundException: org.apache.hive.hcatalog.data.JsonSerDe
                .filter(format -> !"JSON".equals(format))
                .collect(toImmutableSet());
```